concrete/config/coding_style.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+return [
+    'php' => [
+        'filter' => [
+            // Whitepace-separated list of file extensions to be parsed
+            'extensions' => 'php',
+            // Whitelspace-separated list of additional files (relative to the webroot) to be parsed
+            'include' => 'concrete/bin/concrete5',
+        ],
+        'ignore_directories' => [
+            // Whitepace-separated list of directory names that should not be parsed
+            'by_name' => 'vendor',
+            // Whitepace-separated list of directory paths (relative to the webroot) that should not be parsed (allowed placeholders: <HANDLE>)
+            'by_path' => 'application/config/doctrine application/files',
+        ],
+        // Whitepace-separated list of file paths (relative to the webroot) that are executed before checking the PHP version.
+        'bootstrap_files' => 'index.php concrete/dispatcher.php concrete/bin/concrete5',
+        'php_only' => [
+            'non_psr4' => [
+                // Whitepace-separated list of file paths (relative to the webroot) that only contain PHP and that don't follow PSR-4 class names (allowed placeholders: <HANDLE>).
+                'files' => <<<'EOT'
+concrete/attributes/<HANDLE>/controller.php
+application/attributes/<HANDLE>/controller.php
+packages/<HANDLE>/attributes/<HANDLE>/controller.php
+
+packages/<HANDLE>/controller.php
+
+concrete/authentication/<HANDLE>/controller.php
+application/authentication/<HANDLE>/controller.php
+packages/<HANDLE>/authentication/<HANDLE>/controller.php
+
+concrete/blocks/<HANDLE>/controller.php
+application/blocks/<HANDLE>/controller.php
+packages/<HANDLE>/blocks/<HANDLE>/controller.php
+
+concrete/geolocation/<HANDLE>/controller.php
+application/geolocation/<HANDLE>/controller.php
+packages/<HANDLE>/geolocation/<HANDLE>/controller.php
+
+concrete/themes/<HANDLE>/page_theme.php
+application/themes/<HANDLE>/page_theme.php
+packages/<HANDLE>/themes/<HANDLE>/page_theme.php
+EOT
+                ,
+                // Whitepace-separated list of directory paths (relative to the webroot) that contain PHP-only files that don't follow PSR-4 class names (allowed placeholders: <HANDLE>).
+                'directories' => <<<'EOT'
```

Comment:
Is there a reason you chose for a white-spaced list, instead of an array?

Reply:
Overriding arrays in custom configuration files is very hard...